### DISCUSSION TITLE
Community Meeting - August 3, 2024

### DISCUSSION THREAD

**Discussion by wrashi** [GothenburgBitFactory/taskwarrior#3563] [Announcements]
Hey everyone, our next community meeting is going to break our bi-weekly schedule a bit. 
Due to summer schedules, the next one will skip this week and be held next week.

That means our next meeting is 12:00 UTC Saturday, August 3, 2024 ([check that in your time zone @ timeanddate.com](https://www.timeanddate.com/worldclock/meetingtime.html?month=8&day=3&year=2024&p1=5&p2=3904&p3=3918&p4=3921&iv=0)). You can [add this event to your calendar](https://calendar.google.com/calendar/event?action=TEMPLATE&tmeid=NDFvbjIwYnVuZHMwczlxMHRsczZydm9pcnMgd2VzbGV5QG13cGVubmVyLmNvbQ&tmsrc=wesley%40mwpenner.com).

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

The formal agenda portion of the meeting runs ~30 minutes. The room will stay open for anyone who wants to hang out or go deeper into any of the topics from the agenda.

During the community demo portion of this meeting, Dustin will show how to use the new syncing functionality in TaskWarrior 3.x!

**Comment by wrashi** [comment]
Just a note to bring this to top of mind…
Tomorrow, Saturday, August 3 at 12:00 UTC, we'll have our community meeting.

Google Meet joining info
Video call link: https://meet.google.com/tns-suqq-iaj
Or dial: ‪(CA) +1 289-323-1423‬ PIN: ‪830 535 232‬#
More phone numbers: https://tel.meet/tns-suqq-iaj?pin=8726596308824

The formal agenda portion of the meeting runs ~30 minutes. 
The room will stay open for anyone who wants to hang out or go deeper into any of the topics from the agenda.
During the community demo portion of this meeting, Dustin will show off the syncing functionality in TaskWarrior 3.x!

**Comment by wrashi** [comment]
The meeting room is now open for everyone…

**Comment by wrashi** [comment]
The video from today's community meeting is now available. Check it out here…
https://youtu.be/-uWqAZDl830

**Reply by djmitche** [reply]
Interesting discussions today!
